Core/CoreTests/Discussions/DiscussionDetailsViewControllerTests.swift

```diff
@@ -37,7 +37,8 @@ class DiscussionDetailsViewControllerTests: CoreTestCase {
     let webView = MockWebView(features: [])
     class MockWebView: CoreWebView {
         @objc var runningCount = 0
-        override func evaluateJavaScript(_ javaScriptString: String, completionHandler: ((Any?, Error?) -> Void)? = nil) {
+
+        override func evaluateJavaScript(_ javaScriptString: String, completionHandler: (@MainActor (Any?, (any Error)?) -> Void)? = nil) {
```

Comment:
This change results in `Method does not override any method from its superclass` on Xcode 15.4
Same for the change in `DiscussionReplyViewControllerTests`

Reply:
We drop Xcode 15 support.